sol/private/utils.bzl

```diff
@@ -1,4 +1,23 @@
 """Utilities"""
 
 def normalize_version_string(str):
-    return str.replace(".", "_")
\ No newline at end of file
+    return str.replace(".", "_")
+
+def semver_cmp(a, b):
```

Comment:
skylib has this already:
https://github.com/bazelbuild/bazel-skylib/blob/main/lib/versions.bzl#L69

Reply:
Done. 

Thanks, and TIL about int-tuple compatibility with `>=` :)